packages/extension-ui/src/components/RadioGroup.tsx

```diff
@@ -28,9 +36,26 @@ const RadioGroup: React.FC<Props> = ({ className, defaultSelectedValue, onSelect
     [onSelectionChange]
   );
 
+  const alephOptions = withTestNetwork
+    ? options.filter((option) => option.text.includes('Aleph Zero'))
+    : options.filter((option) => option.text === 'Aleph Zero');
+  const otherOptions = options.filter((option) => !option.text.includes('Aleph Zero'));
```

Comment:
This also would be a great use case for reduce since we have only two options - either aleph or not. Should be a simple if that returns both arrays for us rather than two filters :)